migration-engine/connectors/sql-migration-connector/src/sql_schema_differ.rs

```diff
@@ -240,23 +231,36 @@ impl<'schema> SqlSchemaDiffer<'schema> {
         })
     }
 
-    fn drop_foreign_keys<'a>(differ: &'a TableDiffer<'schema>) -> impl Iterator<Item = TableChange> + 'a {
+    fn drop_foreign_keys<'a>(&'a self, drop_foreign_keys: &mut Vec<DropForeignKey>) {
+        for differ in self.table_pairs() {
+            let table_name = differ.previous.name();
+            for dropped_foreign_key_name in differ
+                .dropped_foreign_keys()
+                .filter_map(|foreign_key| foreign_key.constraint_name())
```

Comment:
This is not consistent with `DROP CONSTRAINT`s for primary keys, I think we shouldn't fail to these steps silently if we're missing the constraint name.